Public/Standalone/Set-PDFForm.ps1

```diff
@@ -1,32 +1,62 @@
 function Set-PDFForm {
-    [CmdletBinding()]
```

Comment:
I know I'm asking much but would you be able to provide help? if you type three # in the VSCode it will automatically create help and you just need to provide some details? If not, that's fine.